README.md

```diff
@@ -1,25 +1,33 @@
-### Fetch secrets from AWS Secrets Manager
+# Fetch secrets from AWS Secrets Manager
 
-#### Usage
+---
 
-This small binary fetches JSON formatted secrets from AWS Secrets Manager.  
-The secrets should be in a flat JSON K:V format. 
-The secrets path should usually follow the pattern `$service_name/$env/secrets`  
+## Usage
 
+This small binary fetches JSON formatted secrets from AWS Secrets Manager.  It adds the fetched secrets to the `env` for
+an binary that it executes after retrieving the secrets.
 
-It will use the the IAM role tags to autodiscover the secrets to be fetched, 
-any tags begginig with `secrets_` will be matched, the value should point to the secrets manager path.  
-For example the tag `secrets_default = myservice/dev/secrets` will load the secrets at that location. You can specify multiple tags, the merging order is arbitrary so don't expect precedence.
+The secrets should be stored in a AWS Secrets Manager secret with a value specified in a flat JSON K:V format.
+The Secrets Manager secret name should usually follow the path pattern `$service_name/$env/secrets`.  
 
 
-You then run it like this:  
-```sh
+`fetch-secrets` will use the IAM role tags to auto-discover the secrets to be fetched.
+Any tag keys prefixed with `secrets_` will be matched; the corresponding tag value be a Secrets Manager secret name.  
+eg; The tag `secrets_default = myservice/dev/secrets` will load the secrets stored in the secret named `myservice/dev/secrets`. 
+You can specify multiple tags, however, the merging order is arbitrary so don't expect precedence.
+
+---
+
+## Running
```

Comment:
Some instruction on how to install the `fetch-secrets` binary to the user's Docker container might be useful?